ansible/roles/test/tasks/pfc_wd/functional_test/functional_test.yml

```diff
@@ -57,7 +60,6 @@
   delegate_to: "{{peer_mgmt}}"
   become: true
   when: peer_hwsku | search("Arista") or peer_hwsku | search("arista")
```

Comment:
can we put pfc generator deployment into a single yml file, since it is used in many places?

deploy_pfc_pktgen.yml in the yml,